requirements/base.txt

```diff
@@ -11,7 +11,7 @@ celery==3.1.25            # via -c requirements/constraints.txt, -r requirements
 certifi==2020.6.20        # via requests
 chardet==3.0.4            # via requests
 django-crum==0.7.7        # via edx-proctoring
-django-filter==1.0.4      # via -c requirements/constraints.txt, -r requirements/base.in
+django-filter==1.1.0      # via -c requirements/constraints.txt, -r requirements/base.in
```

Comment:
please use the command `make upgrade`  to update this file

Reply:
with python 3.5 or 2.7?

Reply:
2.7 maybe it's necessary the constraint to 1.1.0